### PR TITLE
[DOC] Documentation

### PR DESCRIPTION
**Pull request by MalinaStefaniaStoicanescu**
Change-Id: I14bc713443ab7563fffe9183168c5ed8b2f3bd5a
Signed-off-by: MalinaStefaniaStoicanescu <malina.stoicanescu@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki [hunk 1]
```diff
@@ -221,11 +234,22 @@ Several Layers are provided for Cybersecurity PAB, PDFB, CDB and ES diagrams. A
 
 [[Image:Images/layers.png]]
```
Comment (pdulth) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki:235]: Can you update also this image with the new Supporting asset layer added

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki [hunk 2]
```diff
@@ -97,7 +107,10 @@ The attributes of a Primary Asset are its corresponding Security Needs, i.e. the
 * Availability: property of being accessible and useable upon demand by an authorized entity.
 * Traceability: property of verifying access and use thanks to documented recorded identification.
 
-An asset can define a value between 0 and 3 for each of these attributes. Project/Product teams shall define the equivalences between these sensitivity levels and those specific to their standards framework.
+By default, an asset can define a value between 0 and 3 for each of these attributes. Project/Product teams shall define the equivalences between these sensitivity levels and those specific to their standards framework.
+If this values are not suitable for the project, the user could customize (remove, add, rename) them from Cybersecurity Configuration.
```
Comment (pdulth) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki:111]: This element is located under the Project element which is hidden by default,

Can you add something like: 

If this values are not suitable for the project, the user could customize (remove, add, rename) them from Cybersecurity Configuration located on the root Project element (see Capella Guide > Preferences > Project Explorer to reveal it)

**Review by pdulth — CHANGES_REQUESTED** [review]

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 3]
```diff
@@ -75,6 +75,18 @@ <h2 id="Threat">Threat</h2>
 				<td>Tampering attacks refer to many forms of sabotage (intentional modification of the information system in a way that makes them harmful).</td>
 			</tr>
 		</table>
+		<p>A 
+			<i>Threat</i> kind could be full customize by the user in Cybersecurity Configuration.
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:79]: The threat kind can be customized by the user in the Cybersecurity Configuration, by editing the Threat.Kind type.

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 4]
```diff
@@ -75,6 +75,18 @@ <h2 id="Threat">Threat</h2>
...
+		</p>
+		<p>
+			<img border="0" src="Images/threat_kind.png"/>
+		</p>
+		<p>The user could delete, edit or remove the threat kind. </p>
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:84]: The user can.

Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:84]: The threat kind can be updated as below:

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 5]
```diff
@@ -258,10 +274,17 @@ <h2 id="Cybersecurity_PAB_Layer_Extensions">Cybersecurity PAB Layer Extensions</
 		<p>
 			<img border="0" src="Images/layers.png"/>
 		</p>
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:276]: Can update this picture to also contain the Supporting Asset Layer?

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 6]
```diff
@@ -258,10 +274,17 @@ <h2 id="Cybersecurity_PAB_Layer_Extensions">Cybersecurity PAB Layer Extensions</
...
+		<p>When the primary asset covers a functional chain, the involved functions and functional exchanges will be highlighted in the same way (if the primary asset is displayed).
+			If FunctionalChains (FC) are displayed in diagram, they will override the colors of Primary Asset Layer!</p>
 		<h3 id="Trust_Boundaries_Layer">Trust Boundaries Layer</h3>
-		<p>The Trust Boundaries Layer will change the diagrams appearence: Trusted components and actors will change their background color to white, untrusted components and actors will change their color to red. Communication edges between Trusted and Untrusted elements, so called Trust Boundaries, will be decorated with a small flame icon:</p>
+		<p>The Trust Boundaries Layer will change the diagrams appearance: Trusted components and actors will change their background color to white, untrusted components and actors will change their color to red. Communication edges between Trusted and Untrusted elements, so called Trust Boundaries, will be decorated with a small flame icon covered by a red border:</p>
 		<p>
-			<img border="0" src="Images/pab_diagram_2.png"/> 
+			<img border="0" src="Images/pab_diagram_2(new).png"/> 
 		</p>
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:288]: Add also in Assets Layer (and mention it in supporting assets layer), that 2 tools are available in this layer, under the Cybersecurity section (show/hide Primary Assets and Threats).

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki (hunk 2 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki]: Malina, add also this part:  located on the root Project element (see Capella Guide > Preferences > Project Explorer to reveal it)

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 7]
```diff
@@ -75,6 +75,18 @@ <h2 id="Threat">Threat</h2>
 				<td>Tampering attacks refer to many forms of sabotage (intentional modification of the information system in a way that makes them harmful).</td>
 			</tr>
 		</table>
+		<p>A 
+			<i>Threat kind</i> can be customized by the user in the Cybersecurity Configuration, by editing the Threat.Kind type.
+		</p>
+		<p>
+			<img border="0" src="Images/threat_kind.png"/>
+		</p>
+		<p>The threat kind can be updated as below: </p>
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:84]: can make to Threat kinds?

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html [hunk 8]
```diff
@@ -75,6 +75,18 @@ <h2 id="Threat">Threat</h2>
 				<td>Tampering attacks refer to many forms of sabotage (intentional modification of the information system in a way that makes them harmful).</td>
 			</tr>
 		</table>
+		<p>A 
+			<i>Threat kind</i> can be customized by the user in the Cybersecurity Configuration, by editing the Threat.Kind type.
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html:79]: threat kinds.

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki [hunk 9]
```diff
@@ -219,13 +232,30 @@ One additional tool is provided by the Viewpoint:
 
 Several Layers are provided for Cybersecurity PAB, PDFB, CDB and ES diagrams. A layer allows to change the presentation of a diagram in order to focus on specific aspects of the modeled system. Layers are activated in the diagram toolbar:
 
-[[Image:Images/layers.png]]
+[[Image:Images/layers(new).png]]
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki:235]: move layesr(new) to layers, do not create (new) files, just update the old ones.

**Review by georgiana-ecobici — APPROVED** [review]

**Review by georgiana-ecobici — APPROVED** [review]

**Review by minhtutonthat — APPROVED** [review]

**Pull request closed without merging**